runtime-modules/forum/src/lib.rs

```diff
@@ -126,6 +138,9 @@ pub trait Config:
     /// Maximum depth for nested categories
     type MaxCategoryDepth: Get<u64>;
 
+    /// Maximum number of category's stickied threads
+    type MaxStickiedThreads: Get<u32>;
+
```

Comment:
There are other limits also, ie.:
```rust
    pub const MaxCategoryDepth: u64 = 6;
    pub const MaxSubcategories: u64 = 40;
    pub const MaxThreadsInCategory: u64 = 20;
    pub const MaxPostsInThread: u64 = 20;
    pub const MaxModeratorsForCategory: u64 = 20;
    pub const MaxCategories: u64 = 40;
```

But since they are not used to limit the size of a bounded collection I haven't changed their type to limit the amount of changes in this PR.